dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-email/src/test/java/org/apache/dolphinscheduler/plugin/alert/email/EmailAlertChannelTest.java

```diff
@@ -107,7 +106,7 @@ public String getEmailAlertParams() {
                 .setValue("dolphinscheduler@126.com")
                 .build();
 
-        PasswordParam mailPassword = PasswordParam.newBuilder(MailParamsConstants.NAME_MAIL_PASSWD, "passwd")
+        InputParam mailPassword = InputParam.newBuilder(MailParamsConstants.NAME_MAIL_PASSWD, "passwd")
                 .setPlaceholder("if enable use authentication, you need input password")
                 .setValue("escheduler123")
```

Comment:
does `escheduler123 ` is a default value? if also we should remove it or change to `dolphinscheduler`

I guess `escheduler` is for `easy scheduler` is pervious name of dolphinscheduler